spec/controllers/statistics_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+RSpec.describe StatisticsController, type: :controller do
+  # let(:statistic) { metric_percentage: 100 }
+
+  describe "GET #count_metric for an used metric" do
+    
+    before :each do
```

Comment:
Why have you opted to use Integers to fill your array instead of MetricConfigurtions and MetricSnapshots built with FactoryGirl?